src/recap.c

```diff
@@ -390,11 +411,70 @@ int process_pcap_stream_state(struct pcap_stream_context *pctx) {
   }
 }
 
+int process_pcap_stream(char *pcap_path, struct pcap_stream_context *pctx) {
+  if (pcap_path != NULL) {
+    if ((pctx->pcap_fd = fopen(pcap_path, "rb")) == NULL) {
+      log_errno("fopen");
```

Comment:
It's always worth being a bit more verbose, especially for errors that are likely to come from user error (e.g. me accidentally writing `recap -p my-file.pcp` instead of `.pcap` or something)!

```suggestion
      log_errno("fopen failed for pcap file %s", pcap_path);
```